doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
All directory names are using the underscore notation, having a `DoctrineMigrations` directory at the root of the project looks weird. What about something like `'%kernel.project_dir%/etc/migrations'` instead?